osc-server/src/main/java/org/osc/core/broker/service/securitygroup/BindSecurityGroupService.java

```diff
@@ -72,30 +76,46 @@
 	@Override
 	public BaseJobResponse exec(BindSecurityGroupRequest request, EntityManager em) throws Exception {
 
-		validateAndLoad(em, request);
+		validateRequest(em, request);
 		UnlockObjectMetaTask unlockTask = null;
 		BaseJobResponse response = null;
+		boolean sfcBind = this.apiFactoryService.supportsNeutronSFC(this.securityGroup) ? true : false;
```

Comment:
boolean sfcBind = this.apiFactoryService.supportsNeutronSFC(this.securityGroup)  this should suffice, no  need to test true or false.

Reply:
modified and addressed this comment.